source/extensions/common/wasm/context.cc

```diff
@@ -270,6 +270,16 @@ WasmResult Context::setTickPeriod(std::chrono::milliseconds tick_period) {
   return WasmResult::Ok;
 }
 
+void Context::onCloseTCP(){
+  if(tcp_connection_closed_){
```

Comment:
Please fix style (also, it looks like this is not cherry-pick of correct commit)?

Reply:
Thanks for catching. Fixed format.

The other commit e64efd960cd8e97287764b9bae39b3a0def95046 is a merge and hence cherry-pick was not picking it.